src/bundle/Resources/public/scss/_balloon-form.scss

```diff
@@ -154,11 +154,6 @@
             }
 
             .ck-toolbar__items {
-                .ck-button {
-                    padding: calculateRem(8px);
```

Comment:
I believe this will change the look of all buttons in the toolbar, I don't think we want to change the design here.

Reply:
@dew326 Hi :) Could please be more precise? How should the design be ? This toolbar is customisable, new buttons can be added on the fly
See image. Thx
![image](https://github.com/ibexa/fieldtype-richtext/assets/61700656/5f8031f8-23ec-45a1-889b-50abb4c540f4)

Reply:
Yeah, screenshot shows exactly what I meant. The spaces between button are smaller after the change.

Reply:
There is no space between table button and the line on the right.
And margin will probably make that space is not clickable, padding is better because the click area is bigger.

Reply:
```scss
                .ck-button {
                    padding: calculateRem(8px);
                    margin: 0;

                    &.ck-insert-table-dropdown-grid-box {
                        padding: 0;
                        margin: var(--ck-insert-table-dropdown-box-margin);
                    }
                }
```
I think this should do the trick to keep current styling of toolbar and fix the table grid.